public_html/index.html

```diff
@@ -14,35 +14,27 @@
 
     <!-- Latest compiled JavaScript -->
     <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
-    
-    <style>
-      body {
-        /* body of document */ 
-        padding-top: 15px;
-      }
-      .img-responsive-vert {
-        /* vertical implementation of bootstrap class img-responsive */
-        width: auto;
-        max-height: 100%;
-      }
-    </style>
+  
+    <!-- main CSS for this site -->
+    <link rel="stylesheet" href="main.css">
   </head>
-  <body>
+  <body >
```

Comment:
Space removed. Committed and pushed.

On 12 July 2016 at 10:25, Alistair Chisholm <notifications@github.com>
wrote:

> In public_html/index.html
> <https://github.com/gordon-qcode/hello-world/pull/6#discussion_r70404531>:
>
> >    </head>
> > -  <body>
> > +  <body >
>
> extra space here not required
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gordon-qcode/hello-world/pull/6/files/6c88dac4459068a357de2efe2f84a0e954fbd550#r70404531>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AS-1f8MLMjNm_OlAusKtRQ88a1-p61aYks5qU113gaJpZM4JGMtG>
> .
>